### PR TITLE
docs: update view session recordings in ui steps

### PR DESCRIPTION
**Pull request by cameronperera**
Since we had steps for viewing session recordings and playback related to the UI already, I split the current steps into a `CLI` tab and `Admin Console` tab. The Admin Console tab is fairly sparse and I wonder if this is an exception for a screenshot or gif showing the player in the UI. Let me know your thoughts on that.

I also wanted to ask if we should include a note for channels that cannot be played back. Right now, they are buttons that show as `View` instead of `Play` and have no player shown and just details related to that specific channel recording.

Example with a screenshot of the player:
Screen Shot 2023-07-12 at 5 31 08 PM

### CODE REVIEW

File: website/content/docs/operations/manage-recorded-sessions.mdx [hunk 1]
```diff
@@ -78,3 +63,38 @@ Substitute the ID of the channel for **chr_1234567890**:
...
+
+You can find all recorded sessions in the UI from the global scope.
+
+1. Log in to Boundary.
+1. Select **Session Recordings** in the navigation pane.
+
+   The **Session Recordings** page displays the created time, user, project,
+   target, and duration of the recording.
+
+## View Session Recording details
```
Comment (Dan-Heath) [inline website/content/docs/operations/manage-recorded-sessions.mdx:80]:
```suggestion
## View session recording details
```

File: website/content/docs/operations/manage-recorded-sessions.mdx [hunk 2]
```diff
@@ -78,3 +63,38 @@ Substitute the ID of the channel for **chr_1234567890**:
...
+
+1. Select **View** next to the session recording you want to view.
+
+   The details page has information related to the session recording and
+   links to related inforation like user, target, and storage bucket.
+
+## Playback Channel Recording
```
Comment (Dan-Heath) [inline website/content/docs/operations/manage-recorded-sessions.mdx:87]:
```suggestion
## Play back channel recording
```

File: website/content/docs/operations/manage-recorded-sessions.mdx [hunk 3]
```diff
@@ -78,3 +63,38 @@ Substitute the ID of the channel for **chr_1234567890**:
...
+
+1. Select **Play** next to the channel recording you want to view.
+
+   The recorded session appears in the media player.
+   You can click the **Play** button located at the bottom of the media player
```
Comment (Dan-Heath) [inline website/content/docs/operations/manage-recorded-sessions.mdx:92]:
```suggestion
   Click the **Play** button located at the bottom of the media player
```

File: website/content/docs/operations/manage-recorded-sessions.mdx [hunk 4]
```diff
@@ -78,3 +63,38 @@ Substitute the ID of the channel for **chr_1234567890**:
...
+   to watch the recording.
+
+   If a recorded session channel does not support playback a **View** button
```
Comment (Dan-Heath) [inline website/content/docs/operations/manage-recorded-sessions.mdx:95]:
```suggestion
   If a recorded session channel does not support playback, a **View** button
```

File: website/content/docs/operations/manage-recorded-sessions.mdx [hunk 5]
```diff
@@ -78,3 +63,38 @@ Substitute the ID of the channel for **chr_1234567890**:
...
+   will be shown. The playback page will display a message stating playback
+   is not supported but still show details specific to that channel.
```
Comment (Dan-Heath) [inline website/content/docs/operations/manage-recorded-sessions.mdx:97]:
```suggestion
   is shown. The playback page displays a message stating that playback
   is not supported, but still shows details specific to that channel.
```

**Review by Dan-Heath — CHANGES_REQUESTED** [review]
I added a couple minor suggestions for style and flow, but otherwise I think this content looks great. Thank you!

**Comment by cameronperera** [comment]
> I added a couple minor suggestions for style and flow, but otherwise I think this content looks great. Thank you!

Thanks for those suggestions @Dan-Heath! They have been applied and ready for review again.

**Review by Dan-Heath — APPROVED** [review]
LGTM, thank you for this @cameronperera !

**Pull request merged**